pkgs/top-level/all-packages.nix

```diff
@@ -6472,6 +6472,8 @@ with pkgs;
 
   recyclarr = callPackage ../tools/video/recyclarr { };
 
+  tabby = callPackage ../development/tools/tabby { };
```

Comment:
Use the new [`pkgs/by-name`](https://github.com/NixOS/nixpkgs/tree/master/pkgs/by-name) format to avoid editing this file